theme/static/js/resourceLandingAjax.js

```diff
@@ -574,18 +574,13 @@ function update_sqlite_file_ajax_submit() {
 }
 
 function get_user_info_ajax_submit(url, obj) {
-    var is_group = false;
     var entry = $(obj).closest("div[data-hs-user-type]").find("#user-deck > .hilight");
-    if (entry.length < 1) {
-        entry = $(obj).parent().parent().parent().parent().find("#id_group-deck > .hilight");
-        is_group = true;
-    }
     if (entry.length < 1) {
         return false;
     }
 
     var userID = entry[0].getAttribute("data-value");
-    url = url + userID + "/" + is_group;
+    url = url + userID + "/false";
```

Comment:
@sblack-usu, the 'is_group' variable wasn't needed in the frontend as you described. I have removed it here, but it seems that the backend expects it still. Once removed from the backend we can get rid of this harcoded  `"/false"` string in the URL.

Reply:
yeah, the backend will still expect it because that endpoint is used for groups as well.

Reply:
Understood, in that case we can leave this as it is.